docs/source/topics/proc_accessing-the-internal-openshift-registry.adoc

```diff
@@ -1,18 +1,18 @@
 [id="accessing-the-internal-openshift-registry_{context}"]
-= Accessing the internal OpenShift registry
+= Accessing the internal {openshift} registry
 
-The OpenShift cluster running in the {prod} instance includes an internal container image registry by default.
+The {ocp} cluster running in the {prod} instance includes an internal container image registry by default.
 This internal container image registry can be used as a publication target for locally developed container images.
-To access the internal OpenShift registry, follow these steps.
+To access the internal {ocp} registry, follow these steps.
 
 .Prerequisites
 
 * {prod} is configured to use the {openshift} preset.
 For more information, see link:{crc-gsg-url}#changing-the-selected-preset_gsg[Changing the selected preset].
 * A running {prod} instance.
 For more information, see link:{crc-gsg-url}#starting-the-instance_gsg[Starting the instance].
-* A working OpenShift CLI ([command]`oc`) command.
-For more information, see link:{crc-gsg-url}#accessing-the-openshift-cluster-with-oc_gsg[Accessing the OpenShift cluster with the OpenShift CLI].
+* A working {openshift} CLI ([command]`oc`) command.
+For more information, see link:{crc-gsg-url}#accessing-the-openshift-cluster-with-oc_gsg[Accessing the {openshift} cluster with the {openshift} CLI].
```

Comment:
was `{ocp} cluster` before in the PR

Reply:
This is due to the titling. We're using `{openshift}` in titles to keep them short, `{ocp}` in the body text of the docs for specificity. The link is titled after the name of the section, which uses the shortened version.